app/src/androidTest/java/com/sdp/movemeet/Activity/ListActivityTest.java

```diff
@@ -0,0 +1,54 @@
+package com.sdp.movemeet.Activity;
+
+import androidx.annotation.NonNull;
+import androidx.test.core.app.ActivityScenario;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.filters.LargeTest;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.FirebaseDatabase;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.CountDownLatch;
+
+@LargeTest
+@RunWith(AndroidJUnit4.class)
+public class ListActivityTest {
+
+    private FirebaseAuth fAuth;
+    private FirebaseDatabase database;
+
+    @Test
+    public void listOfActivityTest() {
```

Comment:
I'm not sure to understand the purpose of this test, what should it be used for ?

Reply:
The class in empty, so the test just launch the class, but I will update this test during my next task